.github/workflows/large-pr-labeler.yml

```
@@ -3,6 +3,11 @@ name: Label Large PR
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    types: [opened, synchronize, reopened]
```

actionlint (1.7.12)

[error] key "types" is duplicated in "pull_request" section. previously defined at line:5,col:5

(syntax-check)